meta-zarhus-bsp/meta-zarhus-bsp-rockchip/conf/machine/quartz64-a.conf

```diff
@@ -0,0 +1,13 @@
+#@TYPE: Machine
+#@NAME: Quartz64 Model A
+#@DESCRIPTION: The Quartz64 Model A is powered by a Rockchip RK3566 quad-core
+# ARM Cortex A55 64-Bit Processor with a Mali G-52 GPU. It comes equipped with
+# 2GB, 4GB or 8GB LPDDR4 system memory, and a 128Mb SPI boot flash.
+# https://pine64.org/devices/quartz64_model_a/
+
+require conf/machine/include/rk3566.inc
+
+KERNEL_DEVICETREE = "rockchip/rk3566-quartz64-a.dtb"
```

Comment:
This board is supported on mainline linux already?

Can this patch go to meta-rockchip as well?

Reply:
This will go there, but firstly I need to check those issues from `dmesg` and try to fix them, this is actually the reason this PR is still in `draft` state.